roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -114,6 +114,10 @@ kubelet_deployment_type: docker
 cert_management: script
 vault_deployment_type: docker
 
+# Enable kubeadm deployment (experimental)
+kubeadm_enabled: false
+kubeadm_token: "change.thisisabadtoken1"
```

Comment:
?

Reply:
I will make a better token generator in a follow up

Reply:
Cool, for now maybe `abcdef.0123456789abcdef` so it at least works?

Reply:
I'll change the default. Ansible doesn't support generating passwords with a mandatory character in a specific place. This is one of the more irritating parts of kubeadm and serves no practical purpose.